ckan/lib/helpers.py

```diff
@@ -2884,3 +2884,8 @@ def make_login_url(
 @core_helper
 def csrf_input():
     return snippet('snippets/csrf_input.html')
+
+
+@core_helper
+def resource_list_view_limit():
+    return int(config.get('ckan.resource_list_view_limit', 8))
```

Comment:
The config declaration will take care of ensuring the type and assigning a default value

```suggestion
    return config['ckan.resource_list_view_limit']
```